gr-analog/grc/analog_probe_avg_mag_sqrd_x.xml

```diff
@@ -43,6 +57,12 @@
 		<name>in</name>
 		<type>$type.input</type>
 	</sink>
+	<source>
+		<name>out</name>
+		<type>$type.output</type>
```

Comment:
Since all options have `<opt>output:float</opt>` you might as well set `<type>float</type>` here